effects/anim-polygons/encode.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+import sys
+import os.path
+import argparse
+from textwrap import TextWrapper
+import xml.etree.ElementTree as ET
+
+xmlns = {"svg": "http://www.w3.org/2000/svg"}
+OFFSET = 10
```

Comment:
Could you explain why do we need the offset?

Reply:
Done!